pages/_app.js

```diff
@@ -2,8 +2,17 @@ import GlobalStyle from "../styles";
 import NavBar from "@/components/Navbar";
 import Header from "@/components/Header";
 import { SWRConfig } from "swr";
+import useSWR from "swr";
 
 export default function App({ Component, pageProps }) {
+  const {
+    data: entries,
+    isLoading,
+    error,
+  } = useSWR("/api/entries", {
+    fallbackData: [],
+  });
```

Comment:
you are not using entries, isLoading or error anywhere in this component. I think you can remove this useSWR hook entirely.